osu.Game.Tests/Visual/SongSelectV2/BeatmapCarouselTestScene.cs

```diff
@@ -454,6 +454,7 @@ public partial class TestBeatmapCarousel : BeatmapCarousel
 
             public Func<List<BeatmapCollection>> AllCollections { get; set; } = () => [];
             public Func<FilterCriteria, Dictionary<Guid, ScoreRank>> BeatmapInfoGuidToTopRankMapping { get; set; } = _ => new Dictionary<Guid, ScoreRank>();
+            public Func<FilterCriteria, Dictionary<Guid, double>> BeatmapInfoGuidToTopRankAccuracyMapping { get; set; } = _ => new Dictionary<Guid, double>();
```

Comment:
I am incredibly reluctant to even approach this PR for two reasons:

- This PR will be immediately and inevitably followed by requests to filter by total score, pp, count of GREATs/GOODs, FCs, etc.
- This approach of spamming dictionaries for every user-score-related filter does not feel like it will scale.

So this PR is opening a whole swath of scope creep that I do not want to spend time burrowed in while other, possibly more important, stuff still needs doing.

Reply:
The current implementation of best rank makes some sense from the angle that it matches the website profile counters of ranks. Accuracy is not used anywhere prominently in the same way, which is why I'm unsure as to what users would want here.

I'm not sure I have an answer for you. This would be something actual users interested in this feature would have to voice their opinion on, somehow.